brewtils/specification.py

```diff
@@ -1,20 +1,27 @@
 # -*- coding: utf-8 -*-
 
-SPECIFICATION = {
+_CONNECTION_SPEC = {
     "bg_host": {
         "type": "str",
-        "description": "The beergarden server FQDN",
+        "description": "Beergarden server FQDN",
         "required": True,
         "env_name": "HOST",
         "alt_env_names": ["WEB_HOST"],
     },
     "bg_port": {
         "type": "int",
-        "description": "The beergarden server port",
+        "description": "Beergarden server port",
         "default": 2337,
         "env_name": "PORT",
         "alt_env_names": ["WEB_PORT"],
     },
+    "bg_url_prefix": {
+        "type": "str",
+        "description": "Beergarden server path",
+        "default": "/",
+        "env_name": "URL_PREFIX",
+        "cli_name": "url_prefix",
+    },
```

Comment:
If someone doesn't change their config and still has 'url_prefix', will this pick it up by default? We should add that to our migration script or this validator.